src/main/kotlin/Archive.kt

```diff
@@ -0,0 +1,55 @@
+class Archive(val name: String, private val notes: MutableList<Note> = mutableListOf()) {
+
+    fun manage() {
+        while (true) {
+            println("\nМеню заметок:")
+            println("0. Создать заметку")
+            for (index in notes.indices) {
+                println("${index + 1}. ${notes[index].title}")
+            }
+            println("${notes.size + 1}. Назад")
+
+            print("Введите номер пункта: ")
+            val input = readLine()
+            val choice = parseInputToInt(input)
+
+            if (choice == null || choice < 0 || choice > notes.size + 1) {
```

Comment:
```suggestion
            if (choice == null || choice !in notes.indices) {
```
Таким образом проверка будет выглядеть более выразительно